### DISCUSSION TITLE
Program Elapsed/Remaining

### DISCUSSION THREAD

**Discussion by bigdmo007** [ekutner/home-connect-hass#106] [Ideas]
Pretty much in line with the category and title.

I moved over from official because of all the non-descript and irrelevant entities. Something that was useful though was outputting program time elapsed and time remaining.

If that's available through the API, brilliant.

Otherwise, a button to forcibly close the door so I didn't need to walk downstairs again would be neat!

**Comment by ekutner** [comment]
The integration automatically creates entities when they are made available by the API. If the remaining time existed in the built-in integration it should be added automatically once you actually start a program that reports that property.
